unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInWorldController.cs

```diff
@@ -429,10 +442,21 @@ public void EnterEditMode()
         RenderSettings.skybox = skyBoxMaterial;
     }
 
+    private void OnAllParcelsFloorLoaded()
+    {
+        biwFloorHandler.OnAllParcelsFloorLoaded -= OnAllParcelsFloorLoaded;
+        initialLoadingController.Hide();
+        inputController.isInputActive = true;
+    }
+
```

Comment:
This is the kind of functionality that can go into the controller (Just read all comments and you will understand this)

Reply:
I have already responded in the previous comment: https://github.com/decentraland/explorer/pull/2227#discussion_r600575647

Thanks for the review, Adri.